### PR TITLE
update about copy

### PR DESCRIPTION
**Pull request by kne42**
([relevant issue](https://airtable.com/tblNxANme1LbVQqq1/viwISga6FERUatih1/recSZReaN0yHXq2V5?blocks=hide))
Screen Shot 2021-06-30 at 9 36 04 AM

### CODE REVIEW

**Comment by kne42** [comment]
just realized this duplicates #119 😳

**Review by neuromusic — APPROVED** [review]

File: client/src/pages/about.mdx [hunk 1]
```diff
@@ -6,6 +6,14 @@ import { LayoutMDX } from '@/components';
 
 **The napari hub is a service of the Chan Zuckerberg Initiative in collaboration with napari.**
 
+## About the napari hub
+
+The **napari hub** is a service of the Chan Zuckerberg Initiative in collaboration with napari.
```
Comment (LCObus) [inline client/src/pages/about.mdx:11]:
```suggestion
The napari hub is a service of the Chan Zuckerberg Initiative in collaboration with napari.
```

File: client/src/pages/about.mdx [hunk 2]
```diff
@@ -14,28 +22,22 @@ imaging data. **napari** is a consensus-based community project and an open
 source tool that enables high performance visualization and exploration of a
 broad range of imaging data, including microscopy, medical imaging, geospatial
 data, and more, with a clearly defined [governance model, mission, and values]({NAPARI_GOVERNANCE}).
-CZI supports development of open source and community
-run tools like **napari**, as part of its mission to accelerate biomedical research
-and help every scientist make progress faster.
+CZI [supports development of open source and community run tools like **napari**]({CZI_EOSS}),
```
Comment (LCObus) [inline client/src/pages/about.mdx:25]:
```suggestion
CZI [supports development of open source and community run tools like napari]({CZI_EOSS}),
```

**Review by LCObus — APPROVED** [review]
I know we bolded napari in the .dev site, but I would suggest we don't here, and remove the bold from the first "napari hub", to maintain consistency.

**Review by codemonkey800 — APPROVED** [review]
🥐

**Pull request merged**